tests/test-amp-style-sanitizer.php

```diff
@@ -120,6 +120,12 @@ public function get_body_style_attribute_data() {
 					'div > span { font-weight:bold; font-style: italic; }',
 				),
 			),
+
+			'col_with_width_attribute' => array(
+				'<table><colgroup><col width="253"/></colgroup></table>',
+				'<table><colgroup><col></colgroup></table>',
+				array(),
+			),
```

Comment:
Please add a test for when a `col` has an existing `style` attribute to make sure its properties don't get clobbered.